api/mapping/mapmodule/logopluginservice.md

```diff
@@ -0,0 +1,21 @@
+# LogoPluginService
+
+## Description
+This service can be used to extend the Logo-plugin.
+
+```javascript
+var info = 'About'
+var link = [
+  {
+    link: 'http://arctic-sdi.org/wp-content/uploads/2016/09/fact-sheet-copy-Sept-2016.pdf',
+    title: 'Arctic SDI Fact Sheet'
+  },
+  {
+    link: 'https://auth0.com/',
+    title: 'Auth0'
+  }
+];
+
+var logoService = Oskari.clazz.create('Oskari.map.LogoPluginService', info, link);
```

Comment:
Service is an API the functionality provides to push more content on the UI in this case.